libraries/websphere_cluster_member.rb

```diff
@@ -20,7 +20,7 @@ class WebsphereClusterMember < WebsphereCluster
     # creates a new profile or augments/updates if profile exists.
     action :create do
       # make sure cluster exists, and we're not adding a server that already exists in a cluster.
-      if cluster_exists?(cluster_name) && !member_of_cluster?(server_name, server_node)
+      if cluster_exists?(cluster_name) && !member?(cluster_name, server_name, server_node)
```

Comment:
So before we were checking if a server was a member of a any cluster, and now we are checking if a server is a member of cluster_name?

Reply:
LGTM otherwise

Reply:
correct. A jvm can only be a member of one cluster, if you try to add server X to two clusters it'll barf and tell you you are nuts...